packages/phone-number-privacy/src/index.ts

```diff
@@ -1,21 +1,148 @@
+import { AttestationsWrapper } from '@celo/contractkit/lib/wrappers/Attestations'
 import { BLINDBLS } from 'bls12377js-blind'
 import * as functions from 'firebase-functions'
+import fetch from 'node-fetch'
 import config, { connectToDatabase } from './config'
+import { ACCOUNTS_COLUMNS, ACCOUNTS_TABLE } from './models/account'
+import { getContractKit } from './web3/contracts'
+
+export const NUM_ATTESTATIONS_REQUIRED = 3
+
+const contractKit = getContractKit()
+const knex = connectToDatabase()
 
 export const getSalt = functions.https.onRequest((request, response) => {
-  const privateKey = new Buffer(config.salt.key)
+  confirmUser()
+  Promise.resolve(getRemainingQueryCount(request.body.account, request.body.phoneNumber)).then(
```

Comment:
Any reason we're using raw promises here instead of making the getSalt handler async?

Reply:
I blame it on my lack of experience with typescript & firebase.  I didn't think it was possible with the firebase functions, but looks like it is
https://www.youtube.com/watch?v=Jr7pDZ1RAUg

I'll update -- that will clean up the code a lot